doc/source/administrator/security.md

```diff
@@ -244,6 +230,51 @@ traditional computing environment), so be very careful.
 
 There's ongoing work on making this easier!
 
+## Audit Cloud Metadata server access
+
+Most cloud providers have a static IP that pods can reach to get metadata about
+the cloud. This metadata can contain very sensitive info and in the wrong hands
+allow attackers to take full control of your cluster and cloud resources. Due to
+this, it is **critical** to secure the metadata service from your user pods that
+could end up running malicious code without knowing it.
+
+The slides beginning at [_Slide
+38_](https://schd.ws/hosted_files/kccncna17/d8/Hacking%20and%20Hardening%20Kubernetes%20By%20Example%20v2.pdf)
+provides more information on the dangers presented by this attack.
+
+This Helm chart blocks access to this metadata in two ways by default, but you
+only need one.
+
+### Block metadata with a NetworkPolicy enforced by a NetworkPolicy controller
+
+If you have _NetworkPolicy controller_ such as Calico in the Kubernetes cluster,
+it will enforce the NetworkPolicy resource created by this chart that blocks
+user access to the metadata server. We recommend relying on this approach if you
+you had a NetworkPolicy controller, and then you can disable the other option.
+
+```yaml
```

Comment:
I don't think you need to include this snippet since it's the default and isn't needed, having the full config here implies it needs to be copied. I've added a suggestion to just mention the config property in-line.

Reply:
Deal!